code/game/objects/items/weapons/storage/med_pouch.dm

```diff
@@ -0,0 +1,202 @@
+/*
+Single Use Emergency Pouches
+ */
+
+/obj/item/storage/med_pouch
+	name = "emergency medical pouch"
+	desc = "For use in emergency situations only."
+	icon = 'icons/obj/med_pouch.dmi'
+	storage_slots = 7
+	w_class = ITEMSIZE_SMALL
+	max_w_class = ITEMSIZE_SMALL
+	icon_state = "pack0"
+	var/opened = FALSE
+	var/open_sound = 'sound/items/rip1.ogg'
+	var/injury_type = "generic"
+	var/static/image/cross_overlay
+	build_from_parts = TRUE
+	worn_overlay = "cross"
+
+	desc_info = "INSTRUCTIONS: 1) Tear open the emergency medical pack using the easy open tab at the top.\
+	2) Carefully remove all items from the pouch and discard the pouch \
+	3) Apply all autoinjectors to the injured party \
+	4) Use bandages to stop bleeding if required.\
+	5) Force the injured party to swallow all pills. \
+	6) Use ointment on any burns if required. \
+	7) Contact the medical team with your location. \
+	8) Stay in place once they respond."
+
+/obj/item/storage/med_pouch/Initialize()
+	. = ..()
+	name = "emergency [injury_type] pouch"
+	make_exact_fit()
+	for(var/obj/item/reagent_containers/pill/P in contents)
+		P.color = color
+	for(var/obj/item/reagent_containers/hypospray/autoinjector/A in contents)
+		A.update_icon()
```

Comment:
Shove in an update_icon() again here so it'll add the cross overlay.

The A.update_icon() is for the autoinjector.

Reply:
This fixed everything, thank you Wezzy.